cli/command/load_balancer.go

```diff
@@ -84,6 +89,11 @@ func (l *LoadBalancerCommand) Command() cli.Command {
 						Value: dhc.UnhealthyThreshold,
 						Usage: "Number of consecutive failures required to count as unhealthy",
 					},
+					cli.StringFlag{
+						Name:  "type",
+						Value: config.DefaultLoadBalancerType,
+						Usage: "Type of load balancer, either 'ELB' or 'ALB' for a classic elastic or application load balancer (default is 'ELB')",
```

Comment:
either 'ELB' for a classic load balancer or....

Reply:
I changed it to "classic or application, load balancer" to be a bit more concise, but if you feel strongly it should be "classic load balancer or application load balancer" I will update it.
✅